docker/dev/prepare_env.sh

```diff
@@ -0,0 +1,8 @@
+#!/usr/bin/env bash
+
+# Install local core and cellprofiler repos
+pip install -e core/
```

Comment:
I think/hope if you `pip install -e` the monorepo right now you get all the necessary subpackages, maybe? Have we checked that @gnodar01 , or do you need to pip install the core and frontend subfolders rn?

Reply:
You currently have to do (from the root of the monorepo):

```
pip install -e src/subpackages/library
pip install -e src/subpackages/core
pip install -e src/frontend
```

I haven't decided on the ideal way to do a top level "install all in developer mode" script. I suppose the easiest way would just be to put a `requirements.txt` at the top level, with:

```requirements.txt
-e src/subpackages/library
-e src/subpackages/core
-e src/frontend
```

and then just run `pip install -r requirements.txt` at the top level.

I wouldn't mind doing that in this PR, with the caveat that it may be subject to change in the near-ish term.